recipes/btk/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "btk" %}
+{% set version = "0.4.dev0" %}
+{% set sha256 = "ee067d0e8f0bd0e4087119c71901db7a453b4ea1bc7e616ddb95d808477d1cd5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Biomechanical-ToolKit/BTKCore/archive/v0.3.zip
+  sha256: {{ sha256 }}
+
+# Patches are applied in bld.bat to handle whitespace issues
+#  patches:
+#    - 0001-Updates-since-3.0.patch
```

Comment:
This is a huge patch. Why is this needed?

Reply:
Good question. BTK is a python 2.7 library which have been heavily used in many biomechanics project. It is also unfortunately abandoned. That is very bad since it is holding many projects back. It is also not trivial to build.

The first patch represents all the work by the maintainer since the 0.3 version in 2014 and until it was abandoned end of 2015. Hence the version 0.4.dev of this recipe. I suspect some of this work was also necessary for building with vc14.  

The second patch is all the fixes I could gather to get Python3 support to work.

It is my hope that this would help a lot of people who don't have the option of  building  BTK them self, and a lot of projects which would otherwise be stuck on Python 2.7.

Reply:
We don't usually package dev releases (we have a few issues discussing this). Is there a way to cut another the release?

It terms of the patch itself, if there a way to put it in a gist and then patch it manually in the build script?

Reply:
> We don't usually package dev releases (we have a few issues discussing this). Is there a way to cut another the release?
I don't think so. The maintainer hasn't responded to anything the last few years. But I will try to reach out by other means than github. Maybe he could tag the latest commit. That would eliminate the large patch. 

> It terms of the patch itself, if there a way to put it in a gist and then patch it manually in the build script?

That is no problem. I will try that.

Reply:
> It terms of the patch itself, if there a way to put it in a gist and then patch it manually in the build script?

I'm sorry, but what does that gain us?

Reply:
Not much, but it does keep the large file out of the staged-recipes git repo.